keps/sig-scheduling/624-scheduling-framework/README.md

```diff
@@ -187,22 +181,31 @@ on a cloned CycleState, and may call those functions more than once before calli
 a specific node.
 
 
-### Filter
+#### Filter
 
 These plugins are used to filter out nodes that cannot run the Pod. For each
 node, the scheduler will call filter plugins in their configured order. If any
 filter plugin marks the node as infeasible, the remaining plugins will not be
 called for that node. Nodes may be evaluated concurrently, and Filter may be called
 more than once in the same scheduling cycle.
 
-### Pre-Score
 
-**Notice: `Pre-Score` is available since v1alpha2, and it's known as `Post-Filter` before this version.**
+#### PostFilter
+
+These plugins are called after Filter phase, but only when no feasible nodes were found
+for the pod. Plugins are called in their configured order. If any PostFilter plugin marks
+the node as Schedulable, the remaining plugins will not be called. A typical PostFilter 
+implementation is preemption, which tries to make the pod schedulable by preempting other Pods.
+
+
+#### PreScore
+
+**Notice: `PreScore` is available since v1alpha2, and it's known as `PostFilter` before this version.**
```

Comment:
the v1alpha2 looks confusing, given that it's not an API. We should say the k8s minor version.

Reply:
I just removed the sentence, the public docs clarifies that, wdyt.

Reply:
sg